### PR TITLE
Heighten amount of Warning around Lets Encrypt Limits

### PR DESCRIPTION
**Pull request by InAnimaTe**
I'm definitely not the first person to run into this and I definitely won't be the last.

Being in a situation in which you've expended all your certificate issuance for a variety of your application domains is extremely frustrating. This may be because you had to relaunch the container multiple times to test new feautures, breakage, or whatever else. When this happens, you either have to change the domain your application is accessed with (and CNAME) or go without SSL/TLS (unless you purchase a cert), which are both terrible options. 

This just bit me today because of the changes @abiosoft made around the tagged images for the last few versions. You can see below that the locations of certificates inside the containers has changed twice in the past 4 versions of caddy. 

* `0.9.0` - `/root/.caddy`
* `0.9.1` - `/home/caddy/.caddy`
* `0.9.2` - `/home/caddy/.caddy`
* `0.9.3` - `/root/.caddy`

Although, I don't understand why this was necessary, he does seem to keep the readme updated. However, this doesn't mean much if the information about which directory cert/keys are kept in is the last entry at the far bottom of the readme ;(

Additionally, I feel that on every tag, a github release should be cut with details about possible differences and things to watch out for. I know this isn't an "official" image but its popular enough (and used in semi-prod envs) to be dangerous if not fully understood.

This PR moves up the information about how to approach bind-mounting this directory and adds a quoted section reminding the user of what could happen. This needs to be at the forefront of every tag to ensure people upgrading in the future don't get bit by this.

### CODE REVIEW

**Comment by InAnimaTe** [comment]
Now that I think about it, it may be worthwhile to actually list the main limits people are going to actually care about i.e. number of certs per week and number of certs from same ip.

File: README.md [hunk 1]
```diff
@@ -12,6 +12,24 @@ $ docker run -d -p 2015:2015 abiosoft/caddy
 
 Point your browser to `http://127.0.0.1:2015`.
 
+> Be aware! If you don't bind mount the location certificates are saved to, you may bypass Lets Encrypt limits
+> rending further certificate generation disallowed! See "Saving Certificates" below!
```
Comment (mholt) [inline README.md]: "bypass" sounds like it won't be a problem; maybe rephrase, something like this:

> ... you could hit your CA's rate limits, preventing obtaining or renewing certificates for a fixed time. ...

Comment (InAnimaTe) [inline README.md]: I agree. I've made some modifications ;)

**Comment by abiosoft** [comment]
@InAnimaTe I had to reverse the change as it is causing too many troubles for users. Sorry.

Since `CADDYPATH` environment variable can be set to override where the certs are stored, I think we can add that to the docs as well. Do you mind adding that ?

Thanks.

**Comment by InAnimaTe** [comment]
Great idea! I've added a blurb about how to do that ;)

**Comment by abiosoft** [comment]
Thanks for this. 👍
